### ISSUE TITLE
documentation issue

### ISSUE THREAD

**Issue by acoul** [interop-dev/django-netjsongraph#13]
FWIW,

following the directions of the README.rst  @ github.com/interop-dev/django-netjsongraph I got issues running ./manage.py migrate

ImportError: No module named netjsongraph_api

copying the configuration from https://pypi.python.org/pypi/django-netjsongraph/0.2.1 got me going:

  Applying django_netjsongraph.0001_initial... OK
  Applying django_netjsongraph.0002_topology_published... OK
  Applying django_netjsongraph.0003_topology_receive_strategy... OK
  Applying django_netjsongraph.0004_increased_addresses_maxlength... OK

**Comment by nemesisdesign** [comment]
Hi @acoul,

which section of the README are you referring to? **Installing for development**?

**Comment by acoul** [comment]
I am referring @ the Setup (integrate in an existing django project)

from django.conf.urls import include, url

from django_netjsongraph.api import urls as netjsongraph_api
from django_netjsongraph.visualizer import urls as netjsongraph_visualizer

urlpatterns = [
    # your URLs ...
    url(r'^api/', include(netjsongraph_api)),
    url(r'', include(netjsongraph_visualizer)),
]

**Comment by nemesisdesign** [comment]
If `from django_netjsongraph.api import urls as netjsongraph_api` fails it means your python installation cannot find *django-netjsongraph*, usually this happens for one of these reasons:

* you have not installed it, have you done the install part?
* you have installed it in a virtualenv but now you have not activated the virtualenv

**Comment by acoul** [comment]
Hi Federico, 

In my first post I meant to say that following the documentation steps from the github.com/interop-dev/django-netjsongraph README.rst resulted in an unsuccessful setup, while following the documentation steps from https://pypi.python.org/pypi/django-netjsongraph/0.2.1 resulted on an operational setup.

I struggled a bit to get it going under apache as I am not an experienced django user, but everything seems to be working quite well.

thank you for creating and sharing with the community such an interesting and useful project.

BTW, we've met at the BattleMesh v5 @ Athens, Greece and I enjoyed back then your presentations.

best regards,

--alex
